src/openapiGenerator/mockGenerator/mockGenerator.ts

```diff
@@ -0,0 +1,47 @@
+import { extractResponses } from "./formatResponse";
+import { readFile, writeFileSync, existsSync, mkdirSync } from "fs";
+import { join } from "path";
+import request from "request";
+
+export function generateMock(swaggerUrl: string, outputPath: string) {
+    if (!!swaggerUrl) {
+        process.env["NODE_TLS_REJECT_UNAUTHORIZED"] = "0";
+        if (swaggerUrl.indexOf("http") > -1) {
+            request.get(swaggerUrl, (error: any, response: any) => {
+                formatJSON(error, response, outputPath);
+            });
+        } else {
+            readFile(swaggerUrl, "utf-8", (error: any, response: any) => {
+                formatJSON(error, response, outputPath);
+            });
+        }
+    }
+}
+
+function formatJSON(error: any, response: any, outputPath: string) {
+    if (error) {
+        throw new Error(error);
+    } else {
+        if (response?.statusCode !== 200) {
+            throw new Error(response.statusMessage);
+        }
+        console.log("################# generating mock ########################");
+        const body = JSON.parse(response.body || response);
+        const extractedBody: any = extractResponses(body);
+        if (!existsSync(outputPath)){
+            mkdirSync(outputPath, { recursive: true });
+        }
+        writeFiles(extractedBody, outputPath);
+        console.log("################# mock generated ########################");
```

Comment:
You can color the success message green for better representation of the state
```suggestion
        console.log("\x1b[31m%s\x1b[0m", "################# Mock generated ########################");
```

Reply:
your suggestion is red not green 
![image](https://user-images.githubusercontent.com/49055952/83118094-7bf5ce00-a100-11ea-87d6-7825f266e521.png)

use `"\x1b[32m%s\x1b[0m"` for green 
![image](https://user-images.githubusercontent.com/49055952/83118336-ce36ef00-a100-11ea-8647-8336472a375b.png)

Reply:
use `\x1b[32m%s\x1b[0m` for green, his example is red

![image](https://user-images.githubusercontent.com/49055952/83118556-181fd500-a101-11ea-9a20-4700b3e7072f.png)

Reply:
updated